packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -304,8 +304,19 @@ export default {
     },
 
     watch: {
-        async authToken () {
-            await this.initialise();
+        /**
+         * Only reload checkout if the token changes as below:
+         * Truthy > Falsey
+         * Falsey > Truthy
+         *
+         * Do not reload checkout if the token changes from one token to another token
+         * as it will always be valid.
+         * */
+        async authToken (newTokenVal, oldTokenVal) {
+            if (!newTokenVal || (!oldTokenVal && newTokenVal)) {
+                await this.initialise();
+                this.setAuthToken(this.authToken);
```

Comment:
This should be outside of the if statement, as we want the store to always have the latest token. This would matter if token changes from truthy to truthy.

Reply:
Yup my bad just realised that too 👍

Reply:
also wouldn't it be better to write the condition as if ((!newTokenVal && oldTokenVal) || (!oldTokenVal && newTokenVal)) 
That would make sure we do not reload when token changes from false to false.

Reply:
Fixed.